File: examples/ruby/organisation/create_organisation.rb
require 'uri'
require 'net/http'

url = URI("https://exenzo.nl/api/v1/organisation")

http = Net::HTTP.new(url.host, url.port)
http.use_ssl = true
http.verify_mode = OpenSSL::SSL::VERIFY_NONE

request = Net::HTTP::Post.new(url)
request["x-authorization"] = 'ORGANISATION_API_TOKEN'
request["content-type"] = 'application/json'
request["cache-control"] = 'no-cache'
request.body = "{\n    \"name\" : \"Test organisatie\",\n    \"image\" : \"/9j/2wCEAAgGBgcGBQgHBwcJCQg....zx8S252ZgUlFJVnOLRRSUAf//Z\"\n}"

response = http.request(request)
puts response.read_body

File: examples/php/organisation/create_organisation.php
<?php

$request = new HttpRequest();
$request->setUrl('https://exenzo.nl/api/v1/organisation');
$request->setMethod(HTTP_METH_POST);

$request->setHeaders(array(
    'cache-control' => 'no-cache',
    'content-type' => 'application/json',
    'x-authorization' => 'ORGANISATION_API_TOKEN'
));

$request->setBody('{
    "name" : "Test organisatie",
    "image" : "/9j/2wCEAAgGBgcGB....ace5r0qMf3dzx8S252ZgUlFJVnOLRRSUAf//Z"
}');

try {
    $response = $request->send();

    echo $response->getBody();
} catch (HttpException $ex) {
    echo $ex;
}